components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -168,6 +171,32 @@ open class GleanInternalAPI {
         }
     }
 
+    /**
+     * Records a list of the currently enabled accessibility services.
+     *
+     * https://developer.android.com/reference/android/view/accessibility/AccessibilityManager.html
+     */
+    private fun recordEnabledAccessibilityServices(applicationContext: Context) {
```

Comment:
Let's make this function `internal` rather than `private`, to make it accessible from tests. Moreover, if instead of passing the `Context` we pass the `accessibilityManager` and `packageManager`, then we could pass in mocks/shadows and unit test this function.

We could additionally make this function return a List, to make it easier to test. We could always set the data in the baseline ping from the call site.